desafio-1/desafio-1.html

```diff
@@ -0,0 +1,10 @@
+<html>
+  <header>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" type="text/css" href="desafio-1.css" />
+  </header>
+  <body>
+  <input class= "input-note" type="text">
```

Comment:
has a space between the class, can be removed

Reply:
thank you i removed rai